lib/hiptest-publisher/options_parser.rb

```diff
@@ -383,10 +395,10 @@ def splitted_files?
   end
 
   def can_name_files?
-    if self[:named_filename]
-      splitted_files? || with_folders?
-    else
+    if filename_pattern.nil?
       false
+    else
+      splitted_files? || with_folders?
```

Comment:
No, because if `filename_pattern.nil?` is true, but `with_folders?` is also true, then we want `can_name_files?` to be false.